src/math/Statistics.cc

```diff
@@ -1029,45 +1013,42 @@ std::pair<double, double> afwMath::Statistics::getResult(
     return ret;
 }
 
-/** @brief Return the value of the desired property (if specified in the constructor)
- */
 double afwMath::Statistics::getValue(
-		afwMath::Property const prop ///< the afw::math::Property to retrieve.
-                                        ///< If NOTHING (default) and you only asked for one
-                                        ///< property in the constructor, that property is returned
+		afwMath::Property const prop
                                      ) const {
     return getResult(prop).first;
 }
 
 
-/** @brief Return the error in the desired property (if specified in the constructor)
- */
 double afwMath::Statistics::getError(
-		afwMath::Property const prop ///< the afw::math::Property to retrieve.
-                                        ///< If NOTHING (default) and you only asked for one
-                                        ///< property in the constructor, that property's error is returned
-                                        ///< \note You may have needed to specify ERROR to the ctor
+		afwMath::Property const prop
                                      ) const {
     return getResult(prop).second;
 }
 
 
 /************************************************************************************************/
-/**
- * Specialisation for Masks; just calculate the "Sum" as the bitwise OR of all pixels
- */
 
 namespace lsst {
 namespace afw {
 namespace math {
 
+/**
+ * Specialisation for Masks; just calculate the "Sum" as the bitwise OR of all pixels
+ *
+ * @param msk Mask whose properties we want
+ * @param msk2 A mask to control which pixels
+ * @param var A variance
+ * @param flags Describe what we want to calculate
+ * @param sctrl Control how things are calculated
+ */
```

Comment:
Should this have been moved to a header?  Is it redundant with docs for the template?